packages/lesswrong/lib/collections/subscriptions/mutations.ts

```diff
@@ -5,6 +5,7 @@ import Users from '../users/collection';
 
 export const defaultSubscriptionTypeTable = {
   "Comments": subscriptionTypes.newReplies,
+  "UserComments": subscriptionTypes.newUserComments,
```

Comment:
We should remove this - this is meant to be a mapping from collection names to subscription types.  Unfortunately that isn't documented here.